Fails to compile on Windows

When compiling on Windows, I get this error:<br><br>

``error[E0600]: cannot apply unary operator `-` to type `u32` ``<br>
in `src/mouse.rs:234:13`<br><br>

Seems to be because `DWORD` is aliased `u32` to which you cannot assign -1<br><br>

rustc : rustc 1.28.0-nightly (5bf68db6e 2018-05-28)<br>
winver: Windows 10 Pro Version 1803 (OS Build 17134.81)

--- a/src/mouse.rs
+++ b/src/mouse.rs
@@ -228,16 +228,17 @@ fn system_toggle(button: Button, down: bool) {
 fn system_scroll(direction: ScrollDirection, clicks: u32) {
     use winapi::um::winuser::{mouse_event, MOUSEEVENTF_WHEEL, WHEEL_DELTA};
     unsafe {
-        let multiplier: DWORD = if direction == ScrollDirection::Up {
-            1
+        let distance: DWORD = WHEEL_DELTA as DWORD * clicks as DWORD;
+        let units: DWORD = if direction == ScrollDirection::Up {
+            distance
         } else {
-            -1
+            std::u32::MAX - (distance - 1)
         };
         mouse_event(
             MOUSEEVENTF_WHEEL,
             0,
             0,
-            WHEEL_DELTA as DWORD * clicks as DWORD * multiplier,
+            units,
             0,
         );
     };

